app/assets/javascripts/common/reports.js

```diff
@@ -1699,3 +1678,21 @@ const intersectDataVerticalLine = {
   },
 };
 
+// ---
+// Object merging functions
+function combineConfigWithBaseConfig(additionalConfig) {
+  return _.mergeWith(
+    baseLineGraphConfig(),
+    additionalConfig,
+    mergeWithCustomizerArrayMerging
```

Comment:
Could we call this something that represents the mode of merging? Maybe `mergeArraysWithConcatenation`?